File: codewars/cpp/remove_string_spaces.cpp
// https://www.codewars.com/kata/57eae20f5500ad98e50002c5/train/cpp


#include <iostream>
#include <regex>

std::string no_space(std::string x) {
    return std::regex_replace(x, std::regex("[\\s]"), "");
}

int main() {
    std::string x;

    getline(std::cin, x);
    std::cout << no_space(x) << "\n";

    return 0;
}

File: codewars/rust/src/remove_string_spaces.rs
//https://www.codewars.com/kata/57eae20f5500ad98e50002c5/train/rust

use std::io;

fn no_space(x : String) -> String{
	x.replace(" ", "").to_string()
}

fn main() {
	let mut x = String::new();

	io::stdin()
		.read_line(&mut x)
		.expect("Failed to read line");

	println!("{}", no_space(x));
}